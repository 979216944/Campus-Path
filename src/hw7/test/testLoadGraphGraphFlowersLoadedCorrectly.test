# Test load a graph flowers correctly.

# Load graph
LoadGraph flowers flowers.tsv

# check graph if it is loaded correctly
ListNodes flowers
ListChildren flowers Aconitum
ListChildren flowers Alchemilla
ListChildren flowers Astilbe
ListChildren flowers Gaillardia
ListChildren flowers Quince
ListChildren flowers Verbena