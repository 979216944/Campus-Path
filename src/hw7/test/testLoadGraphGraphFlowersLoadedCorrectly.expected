# Test load a graph flowers correctly.

# Load graph
loaded graph flowers

# check graph if it is loaded correctly
flowers contains: Aconitum Alchemilla Astilbe Gaillardia Quince Verbena
the children of Aconitum in flowers are: Alchemilla(1.000) Astilbe(1.000) Gaillardia(0.500) Quince(0.500)
the children of Alchemilla in flowers are: Aconitum(1.000) Astilbe(1.000) Gaillardia(1.000) Quince(1.000)
the children of Astilbe in flowers are: Aconitum(1.000) Alchemilla(1.000) Gaillardia(1.000) Quince(1.000)
the children of Gaillardia in flowers are: Aconitum(0.500) Alchemilla(1.000) Astilbe(1.000) Quince(0.500)
the children of Quince in flowers are: Aconitum(0.500) Alchemilla(1.000) Astilbe(1.000) Gaillardia(0.500)
the children of Verbena in flowers are: